plasoscaffolder/frontend/frontend.py

```diff
@@ -0,0 +1,373 @@
+      scaffolder_engine (scaffolder_engine.ScaffolderEngine): the scaffolder
+          engine object, needed to store answers from questions asked.
+
+    Raises:
+      UnableToConfigure: if the answer causes the scaffolder not
+          to be configured properly.
+    """
+    for question in scaffolder.GetQuestions():
+      gather_answer = True
+      # Loop created to give the user a chance to correct each incorrect
+      # answer they provide.
+      while gather_answer:
+        try:
+          value = cls._AskQuestion(question)
+          break
+        except errors.UnableToConfigure as exception:
+          # pylint: disable=assignment-from-no-return
+          cls.OUTPUT_HANDLER.PrintError(
+              'Unable to configure, with error: {0:s}'.format(repr(exception)))
+          gather_answer = cls.OUTPUT_HANDLER.Confirm('Want to try again?')
+          if not gather_answer:
+            raise
+
+      scaffolder_engine.StoreScaffolderAttribute(
+          question.attribute, value, question.TYPE)
+
+  @classmethod
+  def GetDefinition(
+      cls, definition_string: str) -> definition_interface.ScaffolderDefinition:
+    """Return the definition object as chosen by the user.
+
+    Args:
+      definition_string (str): definition name, read from user input.
+
+    Returns:
+      definition_interface.ScaffolderDefinition: the chosen definition object.
+    """
+    definitions = list(
+        definition_manager.DefinitionManager.GetDefinitionNames())
+
+    cls.OUTPUT_HANDLER.PrintNewLine()
+    if not definition_string:
+      cls.OUTPUT_HANDLER.PrintInfo('Available definitions: ')
+      definition_string = cls._GetSelection(definitions, 'Definition')
+
+    if definition_string in definitions:
+      cls.OUTPUT_HANDLER.PrintOutput('{0:s} chosen.'.format(definition_string))
+      def_class = definition_manager.DefinitionManager.GetDefinitionByName(
+          definition_string)
+      return def_class()
+
+    cls.OUTPUT_HANDLER.PrintError(
+        'Definition {0:s} does not exist.'.format(definition_string))
+    return cls.GetDefinition('')
+
+  @classmethod
+  def GetModuleName(cls) -> str:
+    """Return the module name as chosen by the user."""
+    cls.OUTPUT_HANDLER.PrintNewLine()
+    cls.OUTPUT_HANDLER.PrintInfo((
+        'Name of the module to be generated. This can be something like "'
+        'foobar sqlite" or "event analytics".\n\nThis will be used for class '
+        'name generation and file name prefixes.'))
+    return cls.OUTPUT_HANDLER.PromptInfo('Module Name')
+
+  @classmethod
+  def GetProjectPath(
+      cls, definition: definition_interface.ScaffolderDefinition) -> str:
+    """Return the path to the project as chosen by the user.
+
+    Args:
+      definition (definition_interface.ScaffolderDefinition): the chosen
+          definition. Used to validate the project path.
+
+    Returns:
+      str: the path to the project file.
```

Comment:
Does this return the path of the project file, or to its parent folder? Please clarify